### PR TITLE
samples: Bluetooth: direction_finding_connectionless_tx: Update config to allow build for nRF5340 APP core

### PR DESCRIPTION
**Pull request by ppryga-nordic**
Move configuration related with BLE controller into board specific file.
That is required to allow building the sample for nRF5340 where BLE
controller is not part of the sample image.

The PR may be merged when this PR: https://github.com/zephyrproject-rtos/zephyr/pull/36047 is merged into main.

### CODE REVIEW

File: samples/bluetooth/direction_finding_connectionless_tx/boards/nrf52833dk_nrf52833.conf [hunk 1]
```diff
@@ -0,0 +1,9 @@
+CONFIG_BT_CTLR=y
+CONFIG_BT_LL_SW_SPLIT=y
+
+CONFIG_BT_CTLR_ADV_EXT=y
+CONFIG_BT_CTLR_ADV_PERIODIC=y
+
+# Enable Direction Finding TX Feature including AoA and AoD
+CONFIG_BT_CTLR_DF=y
+CONFIG_BT_CTLR_DF_ANT_SWITCH_RX=n
```
Comment (Thalley) [inline samples/bluetooth/direction_finding_connectionless_tx/boards/nrf52833dk_nrf52833.conf:9]: Not sure if important, but it's missing an empty newline at the end

**Review by Thalley — APPROVED** [review]

File: samples/bluetooth/direction_finding_connectionless_tx/README.rst [hunk 2]
```diff
@@ -54,6 +69,11 @@ to enable antenna switching. GPIOS are used by Radio peripheral in order followi
 indices of :code:`dfegpio#-gpios` properties. The order is important because it
 affects mapping of antenna switch patterns to GPIOS (see `Antenna patterns`_).
 
+Atenna matrix configuration for nRF5340DK SOC is part of child image that is run
```
Comment (Thalley) [inline samples/bluetooth/direction_finding_connectionless_tx/README.rst:72]:
```suggestion
Antenna matrix configuration for nRF5340DK SOC is part of child image that is run
```

**Review by Thalley — CHANGES_REQUESTED** [review]

File: samples/bluetooth/direction_finding_connectionless_tx/boards/nrf52833dk_nrf52833.conf (hunk 1 above)
Comment (ppryga) [reply samples/bluetooth/direction_finding_connectionless_tx/boards/nrf52833dk_nrf52833.conf]: New line was added

File: samples/bluetooth/direction_finding_connectionless_tx/README.rst (hunk 2 above)
Comment (ppryga) [reply samples/bluetooth/direction_finding_connectionless_tx/README.rst]: Corrected

**Review by Thalley — APPROVED** [review]

**Comment by ppryga** [comment]
@jhedberg @joerchan could you take a look on the PR?

**Comment by cvinayak** [comment]
Can we avoid turning on `CONFIG_BT_CTLR_ADV_EXT=y`, as this is still experimental (until we complete conformance tests) ?

**Comment by ppryga** [comment]
> Can we avoid turning on `CONFIG_BT_CTLR_ADV_EXT=y`, as this is still experimental (until we complete conformance tests) ?
@cvinayak without advertising extensions the features is not working. Maybe it would be better to put an information about use of experimental features?

**Comment by cvinayak** [comment]
> 
> 
> > Can we avoid turning on `CONFIG_BT_CTLR_ADV_EXT=y`, as this is still experimental (until we complete conformance tests) ?
> 
> @cvinayak without advertising extensions the features is not working. Maybe it would be better to put an information about use of experimental features?

Ok, I will send a PR adding CMake warning message when advertising extensions in Controller is enabled.

**Review by joerchan — APPROVED** [review]

**Review by jhedberg — APPROVED** [review]

**Pull request merged**